HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalSoARecHitsLayerClustersProducer_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+from ..psets.hgcal_reco_constants_cfi import HGCAL_reco_constants as HGCAL_reco_constants
+
+hltHgcalSoARecHitsLayerClustersProducer = cms.EDProducer("HGCalSoARecHitsLayerClustersProducer@alpaka",
+    alpaka = cms.untracked.PSet(
+        backend = cms.untracked.string('')
+    ),
+    hgcalRecHitsSoA = cms.InputTag("hltHgcalSoARecHitsProducer"),
+    deltac = cms.double(1.3),
+    kappa = cms.double(9),
+    outlierDeltaFactor = cms.double(2.0)
+)
```

Comment:
Why the above import of HGCAL_reco_constants which are then not used in this file?